### DISCUSSION TITLE
retryStratery on another node

### DISCUSSION THREAD

**Discussion by 3072141364** [argoproj/argo-workflows#11126] [Q&A]
After a pod in a workflow fails, how can the pod task be retried on other nodes

**Comment by terrytangyuan** [comment]
`retryStrategy.affinity.nodeAntiAffinity`

**Reply by 3072141364** [reply]
my argo version is v3.4.3，I tried "nodeAntiAffinity", but not working as expected  https://github.com/argoproj/argo-workflows/issues/9193
